assume/common/outputs.py

```diff
@@ -212,9 +213,39 @@ async def store_dfs(self):
                 df.to_csv(data_path, mode="a", header=not data_path.exists())
 
             if self.db is not None:
-                df.to_sql(table, self.db.bind, if_exists="append")
+                try:
+                    with self.db.begin() as db:
+                        df.to_sql(table, db, if_exists="append")
+                except ProgrammingError:
+                    self.check_columns(table, df)
+                    # now try again
+                    with self.db.begin() as db:
+                        df.to_sql(table, db, if_exists="append")
+
             self.write_dfs[table] = []
 
+    def check_columns(self, table: str, df: pd.DataFrame):
+        """
+        If a simulation before has been started which does not include an additional field
+        we try to add the field.
+        For now, this only works for float and text.
+        An alternative which finds the correct types would be to use
```

Comment:
What other types there might be? anyway we cannot write dicts or enything else.

Reply:
we do not detect bool or datetime values. But for most use cases it will be enough

Reply:
so maybe we can for now add a raise if the type is not detected? so we see if the problem ever occurs. Or it won't work anyway?

Reply:
If the field is bool (0 or 1) or int, we will add it as float numeric.
Datetime might be added as unix timestamp numeric or raise an error.

This will probably be enough to visualize the values, even if it is not the proper datatype.
And in other cases it will raise anyway. So I don't think we need additional error raising